bot/common/commands.py

```diff
@@ -286,11 +286,45 @@ async def points(
                 ),
             )
 
-        _, metadata = await DisplayPointsStep(
-            guild_id=ctx.guild.id, cache=Redis, bot=bot, channel=ctx.channel, days=days
-        ).send(None, ctx.author.id)
+        embed = discord.Embed(
+            colour=INFO_EMBED_COLOR,
+            title="Your points!",
+            description="Below is a table representation of  "
+            "the points you've accrued from your contributions! ",
+        )
+        try:
+            message = await ctx.response.send_message(
+                "Your points have been sent as a dm!"
+            )
+        except discord.Forbidden:
+            message = await ctx.followup.send(
+                "Please enable DM's in order to use the Govrn Bot!", ephemeral=True
+            )
+            return
 
-        await ctx.response.send_message(metadata.get("msg"), ephemeral=True)
```

Comment:
What is the reason for forcing everything to go via DM rather than an ephemeral message?

Reply:
Can discord not support ephemeral file uploads?

Reply:
ephemeral would work too for file uploads, I can make that change

Reply:
This command can be run from both a DM and within a server correct?

So the ephemeral part is just if they run it in the server correct.  If possible I think it'd be good to keep both options

Reply:
To answer your discord question, if we're able to do both in the DMs and ephemeral, I'd just make the csv automatic if run in the server and still reaction-based if in a DM

Reply:
emoji flow working in DMs, and the ephemeral in the server